client/my-sites/themes/theme-showcase.jsx

```diff
@@ -193,6 +194,12 @@ class ThemeShowcase extends Component {
 		this.setState( { tabFilter: this.tabFilters.ALL } );
 		page( url );
 		this.scrollToSearchInput();
+
+		if ( searchBoxContent ) {
+			this.props.recordTracksEvent( 'calypso_theme_search_performed', {
+				search_query: searchBoxContent,
+			} );
+		}
```

Comment:
Curious, will this trigger multiple times as we type out a search input as the search box content updates as we type? If so should we add some sort of debouncing before sending the event, to avoid spamming tracks with multiple search events for one search action?